2021-offseason/src/main/java/frc/robot/OI.java

```diff
@@ -1,5 +1,15 @@
+package frc.robot;
+
 import edu.wpi.first.wpilibj.XboxController;
-import edu.wpi.first.wpilibj.controller.*;
-public class OI {
+import edu.wpi.first.wpilibj.GenericHID.Hand;
 
-}
\ No newline at end of file
+
+public class OI {
+    public static double getTriggers(XboxController controller){
+        double val = controller.getTriggerAxis(Hand.kRight) - controller.getTriggerAxis(Hand.kLeft);
+        return Math.pow(val, 3);
+    }
+    public static double getLeftStick(XboxController controller){
```

Comment:
line break between 11 and 12
```suggestion
    }
    
    public static double getLeftStick(XboxController controller){
```